src/toast/tests/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-
+  
 # Install the python files
 
 install(FILES
```

Comment:
This list is still missing ops_totalconvolve.py.

Reply:
Done.